### PR TITLE
Model downloaders abstraction

### PR DESCRIPTION
**Pull request by vmarkovtsev**

### CODE REVIEW

File: modelforge/http_.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import logging
```
Comment (zurk) [inline modelforge/http_.py]: `http_.py` to not mix with http package, right?

File: modelforge/http_.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+from modelforge.progress_bar import progress_bar
+
+
+DEFAULT_DOWNLOAD_CHUNK_SIZE = 65536
+
+
+def download(source: str, file: Union[str, BinaryIO], log: logging.Logger,
+             chunk_size: int = -1) -> None:
+    """
+    Download a file from an HTTP source.
```
Comment (zurk) [inline modelforge/http_.py]:
```suggestion
    Download a file from a HTTP source.
```

File: modelforge/http_.py (hunk 1 above)
Comment (vmarkovtsev) [reply modelforge/http_.py]: Yes, otherwise there is a weird import conflict in urllib3

File: modelforge/http_.py (hunk 2 above)
Comment (vmarkovtsev) [reply modelforge/http_.py]: Not really. Pronunciation of "H" starts with "a". Only with abbreviations which start with H.

Comment (zurk) [reply modelforge/http_.py]: ok.

**Pull request merged**